docs/usage samples/didcomm_issue-credential_holder_proposes.mdx

```diff
@@ -92,10 +93,6 @@ payload = {
 <TabItem value="vade_evan_cli" label="vade_evan_cli">
 
 ```sh
-vade_options='{"encryptionKeys":{"encryptionMySecret":"5046adc1dba838867b2bbbfdd0c3423e58b57970b5267a90f57960924a87f156","encryptionOthersPublic":"d92f5eeaa24fd4e66221c770f704a5e2639a476bab82cfec40bd2874abeb481f"}}'
```

Comment:
(This and the following sections)

Why are the values removed here?

Reply:
The values in json args tab are the same values included in the vade_cli tab , i thought it was redundant to have same values there as they have been mentioned in previous tab already ..

Reply:
reverted the changes now